video/video_stream_encoder.cc

```diff
@@ -1467,7 +1471,12 @@ void VideoStreamEncoder::OnFrame(Timestamp post_time,
       kMsToRtpTimestamp * static_cast<uint32_t>(incoming_frame.ntp_time_ms()));
 
   if (incoming_frame.ntp_time_ms() <= last_captured_timestamp_) {
+    // TODO: Check latency mode.
```

Comment:
What did we used to do here?

Reply:
Commented out line 1459-1466, 1480-1489 in old branch.